test/core/event_engine/test_suite/BUILD

```diff
@@ -80,8 +80,11 @@ grpc_cc_test(
     tags = [
         "no_windows",
     ],
-    uses_polling = False,
+    uses_event_engine = True,
+    uses_polling = True,
```

Comment:
same question regarding poller duplication.

Reply:
Yes I wanted to test the posix event engine client and server tests against both types of pollers epoll and poll.